{
  try {
    _executor.execute(task);
    return true;
  }
 catch (  RejectedExecutionException e) {
    LOG.debug(e);
    LOG.warn("Rejected execution of {}",task);
    try {
      if (task instanceof Closeable)       ((Closeable)task).close();
    }
 catch (    Exception x) {
      e.addSuppressed(x);
      LOG.warn(e);
    }
  }
  return false;
}
